modules/runner/src/integrationTest/java/org/apache/ignite/internal/configuration/storage/ItRebalanceDistributedTest.java

```diff
@@ -535,13 +549,102 @@ void testDestroyPartitionStoragesOnRestartEvictedNode(TestInfo testInfo) throws
         checkInvokeDestroyedPartitionStorages(newNode, TABLE_NAME, 0);
     }
 
+    @Test
+    void testRebalanceWithTheSameNodes() throws Exception {
```

Comment:
@sergeyuttsel could you please add the comment about this case? Not for formal reasons, but I'm afraid that this case is a really hard for understanding. You can copy part of my description from our discussions

Reply:
Fixed.